### ISSUE TITLE
client: sendto fail, but just ignore it : No route to host

### ISSUE THREAD

**Issue by nguyenvanquan7826** [EspressifApp/EsptouchForIOS#53]
Hi, I download project and run, but it show error and can not found esp.

```
client: sendto fail, but just ignore it
: No route to host
```

I build to ios 14.5 and xcode 12.5

**Comment by lph65724** [comment]
I have run into the same problem on iOS 14.5.

**Comment by fanbaoying** [comment]
@nguyenvanquan7826 @lph65724  The problem has been resolved, please refer to the latest commit.

**Comment by lph65724** [comment]
＠fanbaoying, will pod install be available on this version?

**Comment by nguyenvanquan7826** [comment]
@fanbaoying  thank, I will try again.

**Comment by fanbaoying** [comment]
@lph65724 The ESPTouch pod version has been updated, the latest version is 1.1.0.
https://github.com/EspressifApps/EspressifTouchSDK

**Comment by lph65724** [comment]
@fanbaoying, thanks, but the available version via cocoapad is still 1.0.3.

**Comment by fanbaoying** [comment]
@lph65724 pod 'EspressifTouchSDK', '1.1.0', Write like this in the Podfile, if you report an error, follow the prompts to solve it. 

out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
run `pod repo update `

**Comment by lph65724** [comment]
@fanbaoying after updating to version 1.1.0, I am still getting the following messages, and ESP32 doesn't receive anything.  Please see the attached Podfile.lock for my Esptouch version
[Podfile.lock.zip](https://github.com/EspressifApp/EsptouchForIOS/files/6499221/Podfile.lock.zip)



client: sendto fail, but just ignore it
: No route to host

**Comment by konewu** [comment]
same problem 

after updating to version 1.1.0, I am still getting the following messages, and ESP32 doesn't receive anything.

PODS:
  - EspressifTouchSDK (1.1.0)
  - IMSLog (1.0.8)
  - IotLinkKit (1.2.1):
    - IMSLog

DEPENDENCIES:
  - EspressifTouchSDK (= 1.1.0)
  - IotLinkKit (= 1.2.1)

**Comment by fanbaoying** [comment]
@konewu 你好：

我看你是在自己的项目 BTLock 上集成的，在 GitHub 公版上是否有这个问题？

有没有尝试过不使用 cocoapods 集成，该用手动导入。

**Comment by huangzhengguo** [comment]
使用 cocoapods 和 手动导入依旧出现同样错误

**Comment by lph65724** [comment]
For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me.  I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code.  Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version.  However, I haven't had time to try cocoapods install yet.  hope this helps

**Comment by huangzhengguo** [comment]
> For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps

Thank you, have solved my issue

**Comment by konewu** [comment]
> @konewu 你好：
> 
> 我看你是在自己的项目 BTLock 上集成的，在 GitHub 公版上是否有这个问题？
> 
> 有没有尝试过不使用 cocoapods 集成，该用手动导入。

你好，大神
我采用手动倒入的方式 可以了，解决了问题， 谢谢🙏

**Comment by vt-vaio** [comment]
> > For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps
> 
> Thank you, have solved my issue

For Objective - C programmers, [self._esptouchTask setPackageBroadcast:true]; will solve the problem too. Thank you so much for the hints.

**Comment by Abhesheksh** [comment]
> For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps

Can you help me with how to 'manually install' the library? which files do I need to put in my project and will I also need to create a bridging header?

**Comment by lph65724** [comment]
> > For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps
> 
> Can you help me with how to 'manually install' the library? which files do I need to put in my project and will I also need to create a bridging header?

Yes, you would need a bridging header if you are using Swift.  Basically, you need to add ESPTouch/ESSPTouchV2 to your own project and write your own EspTouchController in Swift

**Comment by Abhesheksh** [comment]
> > > For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps
> > 
> > 
> > Can you help me with how to 'manually install' the library? which files do I need to put in my project and will I also need to create a bridging header?
> 
> Yes, you would need a bridging header if you are using Swift. Basically, you need to add ESPTouch/ESSPTouchV2 to your own project and write your own EspTouchController in Swift

Dumb question, but I'm a newbie, how do I add the ESPTouch/ESPTouchV2 in my project?

**Comment by lph65724** [comment]
> > > > For those who are still having trouble, EspressifTouchSDK 1.1.0 works for me. I manually installed EspressifTouchSDK v1.1.0 and the same error messages still showed at first, but I realized it's the configuration of EspressifTouch in my own code. Adding task.setPackageBroadcast(true) (Swift) solved it, we didn't need to set broadcast in older version. However, I haven't had time to try cocoapods install yet. hope this helps
> > > 
> > > 
> > > Can you help me with how to 'manually install' the library? which files do I need to put in my project and will I also need to create a bridging header?
> > 
> > 
> > Yes, you would need a bridging header if you are using Swift. Basically, you need to add ESPTouch/ESSPTouchV2 to your own project and write your own EspTouchController in Swift
> 
> Dumb question, but I'm a newbie, how do I add the ESPTouch/ESPTouchV2 in my project?

You just simply just create a folder in your project and drag the files from the sample project into your own.  When Xcode prompts, make sure your select copy item if needed like shown in https://stackoverflow.com/questions/45096410/how-can-i-add-file-to-project-in-xcode-9.
